src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -155,21 +188,35 @@ private static VersionInfo GetVersionInfo()
                 loaderAssemblyName: typeof(ImageLoadingTransformer).Assembly.FullName);
         }
 
-        private protected override IRowMapper MakeRowMapper(DataViewSchema schema) => new Mapper(this, schema);
+        private protected override IRowMapper MakeRowMapper(DataViewSchema schema) => new Mapper(this, schema, Type);
 
         private sealed class Mapper : OneToOneMapperBase
         {
             private readonly ImageLoadingTransformer _parent;
-            private readonly ImageDataViewType _imageType;
+            private readonly bool _type;
 
-            public Mapper(ImageLoadingTransformer parent, DataViewSchema inputSchema)
+            public Mapper(ImageLoadingTransformer parent, DataViewSchema inputSchema, bool type)
                 : base(parent.Host.Register(nameof(Mapper)), parent, inputSchema)
             {
-                _imageType = new ImageDataViewType();
+                _type = type;
                 _parent = parent;
             }
 
             protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, bool> activeOutput, out Action disposer)
+            {
+                disposer = null;
+                // Check for the type of Image, if true load images as VBuffer<bytes> else load images as ImageDataViewType
+                if (_type)
+                {
+                    return MakeGetterVectorDataViewByteType(input, iinfo, activeOutput, out disposer);
+                }
+                else
+                {
+                    return MakeGetterImageDataViewType(input, iinfo, activeOutput, out disposer);
+                }
+            }
```

Comment:
for single lines please do not use "{" braces

Reply:
Just an FYI, this isn't an established coding style requirement. We use https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md as the coding style, and it states:

> We use Allman style braces, where each brace begins on a new line. A single line statement block can go without braces but the block must be properly indented on its own line and must not be nested in other statement blocks that use braces (See issue 381 for examples).

"A single line statement block **can** go without braces", it doesn't say it must go without braces.

If you look at the linked issue from that guide, there are plenty of reasons why people prefer to always use braces. The most famous is the Apple SSL bug - https://github.com/dotnet/corefx/issues/381#issuecomment-144943158.

Reply:
@eerhardt we MUST be consistent when possible. What I’m suggesting is about consistency.

Reply:
If consistency is an absolute requirement, then always using braces is the most consistent approach.

Reply:
Is that the most common pattern in the codebase for single line if statements?

Reply:
I don't know what the most common pattern in this codebase is. But it is definitely the most common pattern in the dotnet foundation.

Check out Roslyn's code style document:

https://github.com/dotnet/roslyn/blob/2e31d973ccfbbf625a1acbb138ce49c9b3ed1554/CONTRIBUTING.md#coding-style

It's example has braces in a single line if statement. It is a well established pattern in the dotnet foundation.

Reply:
I’m talking about this codebase. We need to be consistent in this codebase. Either we changed everywhere in the codebase to use braces or stick with the most consistent pattern in the codebase.